google_helpers/stackdriver.py

```diff
@@ -19,28 +19,13 @@
 from builtins import object
 import logging as logger
 from urllib.parse import quote as urllib2_quote
-
-from oauth2client.service_account import ServiceAccountCredentials
-from googleapiclient import discovery
-from httplib2 import Http
-from .utils import execute_with_retries, build_with_retries
-
+import google.cloud.logging as stackdriver_logging
 
 class StackDriverLogger(object):
-    LOGGING_SCOPES = [
-        'https://www.googleapis.com/auth/cloud-platform',
-        'https://www.googleapis.com/auth/logging.admin',
-        'https://www.googleapis.com/auth/logging.write'
-    ]
 
     def __init__(self, project_name, credentials):
```

Comment:
@wlongabaugh Credentials needs to be dropped, per below

Reply:
Fixed